app/views/classes/_attributive_decisions_panel.html.haml

```diff
@@ -3,13 +3,14 @@
   %h2.fr-col 1. Décisions d'attribution
   = indicator_badge(@attributive_decisions_count, @students_count)
 
-%p La décision d'attribution annuelle rend éligible l'élève à percevoir l'allocation. Ce document doit obligatoirement être transmis à l'élève et conservé par l'établissement en cas de contrôle de la chaîne des dépenses publiques.
+%p La décision d'attribution annuelle rend éligible l'élève à percevoir l'allocation. Ce document doit obligatoirement être transmis à l'élève et conservé toute l'année scolaire et il doit être archivé par l'établissement pendant une durée de 10 ans en cas de contrôle de la chaîne des dépenses publiques.
+
 
 - if @etab.some_attributive_decisions_generating?
   = dsfr_alert(type: :info, title: "Édition des décisions d'attribution en cours", classes: 'fr-mt-3w') do
     %p Les décisions d'attribution sont en train d'être éditées, veuillez rafraichir la page dans quelques minutes. Vous pouvez commencer à remplir des coordonnées bancaires ou des PFMPs pendant ce temps.
 - else
-  %ul.fr-btns-group
+  .buttons-group
```

Comment:
c'est plus une liste ?

Reply:
Non, la liste c'était le markup pour le `btns-list` (qu'on a viré par ce qu'il forçait à foutre des marges sur le label). Là si je laisse la liste j'ai des bullet points devant mes boutons

Reply:
pas de bullet points avec `fr-btns-group`, tu peux pas la garder en continu avec ta nouvelle classe ?